contrib/platform/test/com/sun/jna/platform/win32/VersionHelpersTest.java

```diff
@@ -0,0 +1,66 @@
+package com.sun.jna.platform.win32;
```

Comment:
I missed this in the first round - could you please also the ALv2+LGPL header here? I'd like to converge on that.

Reply:
Sure, will do! I just copied some other random test file that was missing a header so didn't know if that was standard.